pkg/proc/registers_amd64_arch.go

```diff
@@ -7,7 +7,7 @@ import (
 	"golang.org/x/arch/x86/x86asm"
 )
 
-var dwarfToAsm = map[int]x86asm.Reg{
+var dwarfToAsm_amd64 = map[int]x86asm.Reg{
```

Comment:
It looks like we aren't using this file anymore. It was superceded by the RegistersToDwarfRegisters stuff in arch.go. Delete it.

Reply:
modified.